HopsanGUI/CoreAccess.cpp

```diff
@@ -200,6 +200,16 @@ bool CoreSimulationHandler::simulate(const double startTime, const double stopTi
     return gHopsanCore.getSimulationHandler()->simulateSystem(startTime, stopTime, nThreads, coreSystems, modelHasNotChanged, algorithm);
 }
 
+bool CoreSimulationHandler::startRealtimeSimumlation(const double startTime, CoreSystemAccess *pCoreSystemAccess, double realtimeFactor)
+{
+    return gHopsanCore.getSimulationHandler()->startRealtimeSimulation(startTime, pCoreSystemAccess->getCoreSystemPtr(), realtimeFactor);
```

Comment:
Startime was never used, (as far as I could see)

Reply:
See above. I removed it here as well.